Metrics.md

```diff
@@ -0,0 +1,118 @@
+| `factorio_items_launched_total` | force<br/>name                     | How many of each item have been sent to space by each force                                       |
+
+### Trains
+
+Important! Several train metrics have a very high cardinality. This means that you will very likely get many metrics,
+which can slow down your server during metric collection (see `Settings>Scheduler timer` to collect less often) and if
+you do not host your own metric backend (prometheus in our default case), can cost you a lot of money if unchecked! All
+train stats can be disabled in the settings to prevent this.
+
+Histogram bucket sizes can also be adjusted in the settings.
+
+| Name                                                                                                                                                | Labels                   | Description                                                                                    |
+|-----------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------|------------------------------------------------------------------------------------------------|
+| `factorio_train_trip_time`                                                                                                                          | from<br/>to<br/>train_id | How long `train_id` took to go from `from` to `to` in seconds                                  |
+| `factorio_train_wait_time`                                                                                                                          | from<br/>to<br/>train_id | How long `train_id` had to wait while going from `from` to `to` in seconds                     |
+| `factorio_train_trip_time_groups_bucket`<br/>`factorio_train_trip_time_groups_sum`<br/>`factorio_train_trip_time_groups_count`                      | from<br/>to<br/>train_id | Histogram for how long `train_id` took to go from `from` to `to` in seconds                    |
+| `factorio_train_wait_time_groups_bucket`<br/>`factorio_train_wait_time_groups_sum`<br/>`factorio_train_wait_time_groups_count`                      | from<br/>to<br/>train_id | Histogram for how long `train_id` had to wait while going from `from` to `to` in seconds       |
+| `factorio_train_direct_loop_time`                                                                                                                   | a<br/>b                  | ???TODO???                                                                                     |
+| `factorio_train_direct_loop_time_groups_bucket`<br/>`factorio_train_direct_loop_time_groups_sum`<br/>`factorio_train_direct_loop_time_groups_count` | a<br/>b                  | ???TODO???                                                                                     |
```

Comment:
I did not really understand the implementation of these two metrics / what they should represent. They are calculated based on the connection `from-to` for each `train_id`, but `from` and `to` is sorted and `train_id` is dropped, so in direct loops the trains write to the same data point on both `from-to` and `to-from` paths? Also because `train_id` is dropped, this should probably cause problems when one train goes `[A-B-A]` and another goes `[A-B-C-B-A]`, as the latter will have a longer period but still write its period the same gauge data point `A,B`. I guess that would be less of a problem for the histogram, but both the gauge and the histogram seem to be related by name and by implementation, so I did not want to make any assumption when I do not understand the gauge.